contributors.md

```diff
@@ -7,3 +7,4 @@ Includes all git commit authors. Aliases are GitHub user names.
 * Alexandre Berdeaux (FalseIlyu) <ilyulemail@gmail.com>
 * Reinhold Gschweicher (NeroBurner) <pyro4hell@gmail.com>
 * Michał Janiszewski (janisozaur) <janisozaur@gmail.com>
+* Daniele Lombardi (Daniels118)
```

Comment:
As an author it is important to be able to contact you other than discord. Do you have maybe a privacy aware alias?
Needs might be:
* Permission to re-use work.
* Change of license.
* Credit.
* Questions.